src/stakeout.js

```diff
@@ -0,0 +1,39 @@
+import { each } from './underscore_ish';
+
+let handlersByPath = new Map();
+
+function normalize(path) {
+  return Array.isArray(path) ? path.join('.') : path;
+}
```

Comment:
could be helpful in `underscore_ish` which we could rename to `utils` or something simple instead.